bindings/python/crocoddyl/core/activations/quadratic-barrier.cpp

```diff
@@ -35,8 +35,8 @@ void exposeActivationQuadraticBarrier() {
       "The activation is zero when r is between the lower (lb) and upper (ub) bounds, beta\n"
       "determines how much of the total range is not activated. This is the activation\n"
       "equations:\n"
-      "a(r) = 0.5 * ||r||^2 for lb < r < ub\n"
-      "a(r) = 0. for lb >= r >= ub.",
+      "a(r) = 0.5 * ||r||^2 for lb >= r >= ub\n"
```

Comment:
lb means lower bound. So the comparison operator is incorrect.

Reply:
Yes, it should be
```cpp
"a(r) = 0.5 * ||r||^2 for lb > r > ub\n"
"a(r) = 0. for lb <= r <= ub.",
```

Reply:
Done